src/parser/json/file_parser.py

```diff
@@ -0,0 +1,100 @@
+        comment: Optional[str] = file_dict.get("comment")
+        copyright_text: Optional[str] = file_dict.get("copyrightText")
+        file_contributors: List[str] = file_dict.get("fileContributors")
+        file_types: List[FileType] = parse_field_or_log_error(logger=logger, field=file_dict.get("fileTypes"),
+                                                              parsing_method=self.parse_file_types, optional=True)
+
+        license_comments: Optional[str] = file_dict.get("licenseComments")
+
+        license_concluded: Optional[Union[
+            LicenseExpression, SpdxNoAssertion, SpdxNone]] = parse_field_or_log_error(logger=logger,
+                                                                                      field=file_dict.get(
+                                                                                          "licenseConcluded"),
+                                                                                      parsing_method=self.license_expression_parser.parse_license_expression,
+                                                                                      optional=True)
+
+        license_info_in_files: Optional[
+            Union[List[
+                LicenseExpression], SpdxNoAssertion, SpdxNone]] = parse_field_or_log_error(
+            logger=logger, field=file_dict.get("licenseInfoInFiles"),
+            parsing_method=self.license_expression_parser.parse_license_expression, optional=True)
+        notice_text: Optional[str] = file_dict.get("noticeText")
+        raise_parsing_error_if_logger_has_messages(logger, f"file {name}")
+
+        file = construct_or_raise_parsing_error(File, dict(name=name, spdx_id=spdx_id, checksums=checksums,
+                                                           attribution_texts=attribution_texts, comment=comment,
+                                                           copyright_text=copyright_text, file_type=file_types,
+                                                           contributors=file_contributors,
+                                                           license_comment=license_comments,
+                                                           concluded_license=license_concluded,
+                                                           license_info_in_file=license_info_in_files,
+                                                           notice=notice_text)
+                                                )
+        return file
+
+    @staticmethod
+    def parse_file_types(file_types_list: List[str]) -> List[FileType]:
+        logger = Logger()
+        file_types = []
+        for file_type in file_types_list:
+            try:
+                file_type = FileType[file_type]
+            except KeyError:
+                logger.append(f"FileType {file_type} is not valid.")
+                continue
+            file_types.append(file_type)
+        raise_parsing_error_if_logger_has_messages(logger, "file_types")
```

Comment:
True, I will change this so that the string is uppercase and matches the corresponding class name or lowercase if the message doesn't belong to one explicit class.